data/strings/strings.txt

```diff
@@ -16446,6 +16446,96 @@
     zh-Hans = 运营者
     zh-Hant = 營運者
 
+  [editor_category_unsuitable_title]
+    tags = android
+    en = Can't find a suitable category?
+    af = Kan u nie 'n geskikte kategorie vind nie?
+    ar = لا يمكن العثور على فئة مناسبة؟
+    az = Uyğun kateqoriya tapa bilmirsiniz?
+    be = Не можаце знайсці прыдатную катэгорыю?
+    bg = Не можете да намерите подходяща категория?
+    ca = No trobeu una categoria adequada?
+    cs = Nemůžete najít vhodnou kategorii?
+    da = Kan du ikke finde en passende kategori?
+    de = Keine passende Kategorie gefunden?
+    el = Δεν μπορείτε να βρείτε την κατάλληλη κατηγορία;
+    es = ¿No halla una categoría adecuada?
+    et = Ei leia sobivat kategooriat?
+    eu = Ezin duzu kategoria egokirik aurkitu?
+    fa = آیا نمی توانید یک دسته بندی مناسب پیدا کنید؟
+    fi = Etkö löydä sopivaa luokkaa?
+    fr = Tu ne trouves pas de catégorie appropriée ?
+    he = לא מוצאים קטגוריה מתאימה?
+    hi = कोई उपयुक्त श्रेणी नहीं मिल रही?
+    hu = Nem találja a megfelelő kategóriát?
+    id = Tidak dapat menemukan kategori yang sesuai?
+    it = Non riesci a trovare una categoria adatta?
+    ja = 適切なカテゴリーが見つからない？
+    ko = 적합한 카테고리를 찾을 수 없나요?
+    lt = Nerandate tinkamos kategorijos?
+    mr = योग्य श्रेणी शोधू शकत नाही?
+    nb = Finner du ikke en passende kategori?
+    nl = Kun je geen geschikte categorie vinden?
+    pl = Nie możesz znaleźć odpowiedniej kategorii?
+    pt = Não encontra uma categoria adequada?
+    pt-BR = Você não encontra uma categoria adequada?
+    ro = Nu găsiți o categorie potrivită?
+    ru = Не можешь найти подходящую категорию?
+    sk = Nemôžete nájsť vhodnú kategóriu?
+    sv = Kan du inte hitta en lämplig kategori?
+    sw = Je, huwezi kupata aina inayofaa?
+    th = ไม่พบหมวดหมู่ที่เหมาะสมใช่ไหม?
+    tr = Uygun bir kategori bulamıyor musunuz?
+    uk = Не можете знайти відповідну категорію?
+    vi = Không thể tìm thấy một danh mục phù hợp?
+    zh-Hans = 找不到合适的类别？
+    zh-Hant = 找不到合適的類別？
+
+  [editor_category_unsuitable_text]
+    tags = android
+    en = Organic Maps allows to add simple point categories only, that means no towns, roads, lakes, building outlines, etc. Please add such categories directly to <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Check our <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> for detailed step by step instructions.
```

Comment:
There is already "Categories?" mentioned in the title, no need to mention it here. We can actually include all 3 forms: categories + object types + features, bingo!
Also a few more improvements:


`en = Organic Maps allows adding only simple object types, which means no towns, roads, lakes, building outlines, etc. Please add or edit more complex features at <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Check our <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">guide</a> for detailed step-by-step instructions and other recommended map editing apps.`

Reply:
> We can actually include all 3 forms: categories + object types + features, bingo!

Please not, that's just confusing

Reply:
For the future it would be good if you could at some point indicate that the English string is good and ready for translation reviews. It's not good that the translator now have to translate similar string over and over again.

Reply:
Sorry, it is not enough time to manage that amount of PRs alone. En texts can be pre-approved before anything else next time, even in the chat.

Reply:
another option is make separate translation PRs after the main one is merged (incl. en strings) - it'll reduce amount of non-relevant notifications for translators too

the downside is that its extra work and surely it'll get forgotten sometimes

Reply:
(and with Weblate it'll be a non-issue)